packages/error-reporting/system-test/testAuthClient.js

```diff
@@ -31,9 +31,10 @@ var forEach = require('lodash.foreach');
 var assign = require('lodash.assign');
 var pick = require('lodash.pick');
 var omitBy = require('lodash.omitby');
+var util = require('util');
 
 const ERR_TOKEN = '_@google_STACKDRIVER_INTEGRATION_TEST_ERROR__';
-const TIMEOUT = 20000;
+const TIMEOUT = 30000;
```

Comment:
Was this needed just for extra assurance or does this change increase the time taken somehow?